src/css/base.scss

```diff
@@ -2260,6 +2260,10 @@ textarea,
     min-height: 28px;
 }
 
+.mh-24 {
```

Comment:
Please move this class above `mh-28`

Reply:
Add a request please add comment from where the `minimum height` section is being started

Reply:
Sure will do that 👍

Reply:
Updated the css as Sohel suggested. Please have a look